### ISSUE TITLE
The information provided on mouse hover is not accessible for keyboard-only users: A11y_.NET 9 template_.NET 9 template_Home_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26799]
### GitHub Tags
#A11yTCS; #A11ySev2; #MAC; #WCAG2.1.1; #SH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; #Keyboard; #A11yMAS;#AILimited;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Observe that information provided on mouse hover is not accessible for keyboard-only users.

### Actual Result:
The information provided on mouse hover is not accessible for keyboard-only users.

### Expected Result:
The information displayed on mouse hover should also be accessible to keyboard-only users.

### User Impact:
keyboard users will not be able to get the chart data and won't be able to understand the purpose.

### Attachment:
 
https://github.com/user-attachments/assets/1665a921-08f2-4613-b616-fd10822153fa

**Comment by InduPriya1805** [comment]
Rev:InSe

**Comment by sheiksyedm** [comment]
Duplicate #26234

**Comment by tj-devel709** [comment]
Fix is in this PR: https://github.com/dotnet/maui-samples/pull/569
